src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/create-your-own-manifest.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Create your own manifest file
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Installation
+metaDescription: How to create a manifest using Helm as a templating system
+---
```

Comment:
Support engineers pointed us that it is difficult to set things between different integrations because we have really different UX between integrations, like the Proxy example that I pointed out in the docs.

The migration guide we have right now is meant to be linked in the guided install, the guided install has a manifest renderer so we have to provide a way to create manifests on demand.

I do not like it but we have to. Could you provide any other ideas?

Reply:
@mangulonr What are your thoughts about this? I think adding to our docs things we do not support, saying we do not support them, will likely bring more trouble than what's worth.